deckhand/tests/unit/db/base.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import testtools
```

Comment:
I don't think it should be changed for this PR, but we probably need to coalesce on a single testing framework so developers have an easier time moving between UCP projects. Drydock uses `pytest`, not sure what else is in use.